docs/source/index.rst

```diff
@@ -26,15 +26,14 @@ PyTorch is an optimized tensor library for deep learning using GPUs and CPUs.
    packages
 
 .. toctree::
+   :glob:
    :maxdepth: 1
    :caption: Python API
 
    torch
-   nn
+   nn/index.rst
```

Comment:
(not blocking for this pr) It would be nice if someone could click on torch.nn and have it expand to its contituents, line what "Notes" does today.

Notes (with expandable triangle):
![image](https://user-images.githubusercontent.com/5652049/76989620-09f15380-691d-11ea-93c6-e17d6de830ba.png)

torch.nn (in this PR)
![image](https://user-images.githubusercontent.com/5652049/76989639-11186180-691d-11ea-8851-27fc935a48f0.png)

How hard would it be to implement this?